doc/source/serve/model-multiplexing.md

```diff
@@ -0,0 +1,70 @@
+# Model Multiplexing
+
+This section helps you understand how to write multiplexed deployment by using the `serve.multiplexed` and `serve.get_multiplexed_model_id` APIs.
+
+This is an experimental feature and the API may change in the future. You are welcome to try it out and give us feedback!
+
+## Why model multiplexing?
+
+Model multiplexing is a technique for serving multiple models in a single replica. Traffic is routed to the corresponding model based on the request header. To serve multiple models with a pool of replicas to optimize cost. This is useful in cases where you might have many models with the same shape but different weights that are sparsely invoked. If any replica for the deployment has the model loaded, incoming traffic for that model (based on request header) will automatically be routed to that replica avoiding unnecessary load time.
```

Comment:
Model multiplexing is a technique for serving multiple models in a single replica. -> This is not entirely accurate right? It's to serve multiple similar models with a pool of replicas.

Reply:
rephrased.